project/accounts/views.py

```diff
@@ -178,12 +179,14 @@ class UserProfileView(LoginRequiredMixin, View):
 
     def get(self, request, username=None):
         profile = get_object_or_404(Profile, user__username=username)
+        categories = profile.categories.all()
```

Comment:
Oh that's fine @brylie but, I already performed this logic at the backend or would you prefer I do it inside the HTML syntax?